algoliasearch-java-net/src/main/java/com/algolia/search/JavaNetHttpRequester.java

```diff
@@ -40,12 +46,17 @@ public JavaNetHttpRequester(@Nonnull ConfigBase config) {
         HttpClient.newBuilder()
             .executor(config.getExecutor())
             .version(HttpClient.Version.HTTP_2)
+            .sslParameters(sslParametersCiphers())
```

Comment:
Does `sslParameters` append to the default value of the JVM or does it completely override them?

Reply:
After checking it is overriding them; and I've discovered another workaround: setting `SSLParameters` protocols to `null`! 

This made me think of another approach to do this: 
since this issue is related to a specific Java version (_Java 11 up until 11.0.14, and might be fixed later_), we can expose a customizable `HttpClient.Builder` (like for apache http client), this will allow the users to implement workarounds for this issue and any future ones, introduce the workaround for the  Java 11 tests, and run test against latest version (Java 17 for example).

wdyt ?

Reply:
Sounds like a great idea to me :)

Reply:
Nice! I will close this PR then, I've opened another one instead.